### PR TITLE
Tng 1141 remove max boundaries of caas pagination

### PR DESCRIPTION
**Pull request by christian-breidler**

### CODE REVIEW

File: src/modules/FSXAProxyApi.ts [hunk 1]
```diff
@@ -138,27 +138,16 @@ export class FSXAProxyApi implements FSXAApi {
     if (pagesize < 1) {
       this._logger.warn(
         'fetchByFilter',
-        `Given pagesize: ${pagesize} was smaller than 1, pagesize will be set to 1`
+        'pagesize must be greater then zero! Using fallback of 30.'
       )
       pagesize = 1
```
Comment (boersteken) [inline src/modules/FSXAProxyApi.ts:143]: Please adjust this pagesize fallback accordingly (and fix typos of warnings).

File: src/modules/FSXARemoteApi.ts [hunk 2]
```diff
@@ -464,6 +458,12 @@ export class FSXARemoteApi implements FSXAApi {
         case 401:
           throw new Error(FSXAApiErrors.NOT_AUTHORIZED)
         default:
+          if (response.status === 400) {
```
Comment (boersteken) [inline src/modules/FSXARemoteApi.ts:461]: To be able to handle this error in our Nuxt stack (FSXAProxyApi) we should probably throw a specific Error for general client errors (400 responses) and include the response body. We can then catch and include it in the response for the original request of the FSXAProxyApi.

**Review by boersteken — CHANGES_REQUESTED** [review]

File: src/modules/FSXAProxyApi.ts (hunk 1 above)
Comment (webgyver) [reply src/modules/FSXAProxyApi.ts]: Thanks, fixed.

File: src/modules/FSXARemoteApi.ts (hunk 2 above)
Comment (boersteken) [reply src/modules/FSXARemoteApi.ts:461]: We decided to skip this.

**Pull request merged**